frontends/bmd/src/app.test.tsx

```diff
@@ -72,6 +74,38 @@ it('uses kiosk storage when in kiosk-browser', async () => {
   );
   await advanceTimersAndPromises();
   expect(kiosk.storage.get).toHaveBeenCalled();
+  delete window.kiosk;
+});
+
+it('changes screen reader settings based on keyboard inputs', async () => {
+  const mockTts = fakeTts();
+  const screenReader = new AriaScreenReader(mockTts);
+  jest.spyOn(screenReader, 'toggle');
+  jest.spyOn(screenReader, 'changeVolume');
+
+  render(
+    <App
+      screenReader={screenReader}
+      card={new MemoryCard()}
+      machineConfig={fakeMachineConfigProvider()}
+    />
+  );
+
+  await advanceTimersAndPromises();
+
+  // check that 'r' toggles the screen reader
+  expect(screenReader.toggle).toHaveBeenCalledTimes(0);
+  fireEvent.keyDown(screen.getByText('Device Not Configured'), { key: 'r' });
+  await waitFor(() => {
+    expect(screenReader.toggle).toHaveBeenCalledTimes(1);
+  });
+
+  // check that 'F17' changes volume
+  expect(screenReader.changeVolume).toHaveBeenCalledTimes(0);
+  fireEvent.keyDown(screen.getByText('Device Not Configured'), { key: 'F17' });
```

Comment:
If there's any way to not depend on the a particular piece of text here that is irrelevant to the action, that will make this test a little bit more robust if we choose to change the layout / wording for the "Device Not Configured" screen. Maybe add something with a `test-id`, and use `getByTestId` ?

Not strictly required to merge, just a suggestion.

Reply:
Good call, just found I can use `getByRole('main')` which is a degree more resilient.